reference/usage-billing.mdx

```diff
@@ -44,7 +44,7 @@ To purchase compute credits:
 Setting a monthly spending limit allows you to control costs.
 
 <Warning>
-The spending limit takes into account the costs you’re billed for at the end of your billing period. If your organization reaches the spend limit in a billing period, Axiom immediately prevents actions that generate further usage costs. For example, you can’t run queries, you can’t ingest more data, and your monitors can’t query your data. This can cause interruptions to your service.
+The spending limit takes into account the costs you’re billed for at the end of your billing period. If your organization reaches the spend limit in a billing period, Axiom immediately prevents actions that generate further costs. For example, if you have used up the allowances included in your pricing plan, you can’t run queries or ingest more data, and your monitors can’t query your data. This can cause interruptions to your service.
```

Comment:
/s/if you have _used up_ the allowances/if you have _consumed_ the allowances

"consumed" is better than "used up", wdyt ?